lib/ansible/plugins/become/pbrun.py

```diff
@@ -95,7 +95,7 @@ def build_become_command(self, cmd, shell):
 
         become_exe = self.get_option('become_exe') or self.name
         flags = self.get_option('become_flags') or ''
-        user = self.get_option('become_user')
+        user = self.get_option('become_user') or ''
```

Comment:
actually, the correct way to handle this is adding `default: ''` to the `DOCUMENTATION` entry for `become_user`, the same goes for existing `or ''`